Reinstate "-dev" suffix for 2.4.4

gdemonet opened the pull request:
We chose to set "-discontinued" since `development/2.4` is not being
actively worked on anymore.
This however has an unusual meaning, which is even false: "the version
2.4.4 will **never** be released".

Completely removing a version will be achieved by removing the
corresponding `development/` branch. For now, since we may need an
upgrade path from 1.x to 2.x, we will keep 2.0 to 2.4 open.

bert-e commented:
# Hello gdemonet,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://eve.devsca.com/github/scality/metalk8s/bert-e/doc/user).





*Status report is not available.*

bert-e commented:
# Conflict




A conflict has been raised during the creation of
integration branch `w/2.5/bugfix/reinstate-dev-suffix-for-24` with contents from `bugfix/reinstate-dev-suffix-for-24`
and `development/2.5`.

**I have not created the integration branch.**


Here are the steps to resolve this conflict:

```bash
 $ git fetch
 $ git checkout -B w/2.5/bugfix/reinstate-dev-suffix-for-24 origin/development/2.5
 $ git merge origin/bugfix/reinstate-dev-suffix-for-24
 $ # <intense conflict resolution>
 $ git commit
 $ git push -u origin w/2.5/bugfix/reinstate-dev-suffix-for-24
```

gdemonet commented:
/approve

bert-e commented:
# Integration data created



I have created the integration data for the additional destination branches.

* this pull request will merge `bugfix/reinstate-dev-suffix-for-24` into
`development/2.4`
* [w/2.5/bugfix/reinstate-dev-suffix-for-24](https://github.com/scality/metalk8s/compare/development/2.5...w/2.5/bugfix/reinstate-dev-suffix-for-24) will be merged into `development/2.5` 
* [w/2.6/bugfix/reinstate-dev-suffix-for-24](https://github.com/scality/metalk8s/compare/development/2.6...w/2.6/bugfix/reinstate-dev-suffix-for-24) will be merged into `development/2.6` 



The following branches will **NOT** be impacted:

* `development/1.0`
* `development/1.1`
* `development/1.2`
* `development/1.3`
* `development/2.0`
* `development/2.1`
* `development/2.2`
* `development/2.3`




You can set option `create_pull_requests` if you need me to create
**integration pull requests** in addition to integration branches, with:

```
@bert-e create_pull_requests
```







*The following options are set:* **approve**

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @nootal

* @alexandre-allard-scality

* @lucieleonard

* @sayf-eddine-scality

* @slaperche-scality

* @nicolast

* @gdemonet

* @carlito-scality

* @thomasdanan

* @jbertran

* @teddyandrieux

* @cmonfort

* @monpote










*The following options are set:* **approve**

bert-e commented:
# Build failed



The build for [commit](https://github.com/scality/metalk8s/commit/97a9a0c4c611c4a94de4688e018c1e2c65fdb25e) did not succeed in branch bugfix/reinstate-dev-suffix-for-24.




*The following options are set:* **approve**

gdemonet committed:
Reinstate "-dev" suffix for 2.4.4

We chose to set "-discontinued" since development/2.4 is not being
actively worked on anymore.
This however has an unusual meaning, which is even false: "the version
2.4.4 will never be released".

Completely removing a version will be achieved by removing the
corresponding development/ branch. For now, since we may need an
upgrade path from 1.x to 2.x, we will keep 2.0 to 2.4 open.

bert-e commented:
# History mismatch



Merge commit #97a9a0c4c611c4a94de4688e018c1e2c65fdb25e on the integration branch
`w/2.5/bugfix/reinstate-dev-suffix-for-24` is merging a branch which is neither the current
branch `bugfix/reinstate-dev-suffix-for-24` nor the development branch
`development/2.5`.

It is likely due to a rebase of the branch `bugfix/reinstate-dev-suffix-for-24` and the
merge is not possible until all related `w/*` branches are deleted or updated.

**Please use the `reset` command to have me reinitialize these branches.**





*The following options are set:* **approve**

gdemonet commented:
/after_pull_request=2433

bert-e commented:
# Waiting for other pull request(s)



The current pull request is locked by the **after_pull_request** option.

In order for me to merge this pull request, run the following actions first:


:arrow_right: Merge the `OPEN` pull request:

* pull request #2433





Alternatively, delete all the **after_pull_request** comments from this pull request.





*The following options are set:* **approve, after_pull_request**

gdemonet commented:
/reset

bert-e commented:
# Reset complete



I have successfully deleted this pull request's integration branches.






*The following options are set:* **approve, after_pull_request**

bert-e commented:
# Waiting for other pull request(s)



The current pull request is locked by the **after_pull_request** option.

In order for me to merge this pull request, run the following actions first:


:arrow_right: Merge the `OPEN` pull request:

* pull request #2433





Alternatively, delete all the **after_pull_request** comments from this pull request.





*The following options are set:* **approve, after_pull_request**

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @nootal

* @alexandre-allard-scality

* @lucieleonard

* @sayf-eddine-scality

* @slaperche-scality

* @nicolast

* @gdemonet

* @carlito-scality

* @thomasdanan

* @jbertran

* @teddyandrieux

* @cmonfort

* @monpote










*The following options are set:* **approve, after_pull_request**

bert-e commented:
# Build failed



The build for [commit](https://github.com/scality/metalk8s/commit/a682a429fee342cebe93b0457a2d0c0e042f1868) did not succeed in branch w/2.5/bugfix/reinstate-dev-suffix-for-24.




*The following options are set:* **approve, after_pull_request**

bert-e commented:
# Build failed



The build for [commit](https://github.com/scality/metalk8s/commit/27353864fa1e939c6ea9a6c8ef50089386f653f7) did not succeed in branch w/2.6/bugfix/reinstate-dev-suffix-for-24.




*The following options are set:* **approve, after_pull_request**

bert-e commented:
# In the queue



The changeset has received all authorizations and has been added to the
relevant queue(s). The queue(s) will be merged in the target development
branch(es) as soon as builds have passed.

The changeset will be merged in:


* :heavy_check_mark: `development/2.4`

* :heavy_check_mark: `development/2.5`

* :heavy_check_mark: `development/2.6`



The following branches will **NOT** be impacted:

* `development/1.0`
* `development/1.1`
* `development/1.2`
* `development/1.3`
* `development/2.0`
* `development/2.1`
* `development/2.2`
* `development/2.3`



There is no action required on your side. You will be notified here once
the changeset has been merged. In the unlikely event that the changeset
fails permanently on the queue, a member of the admin team will
contact you to help resolve the matter.

*IMPORTANT*

Please do not attempt to modify this pull request.

* Any commit you add on the source branch will trigger a new cycle after the
  current queue is merged.
* Any commit you add on one of the integration branches will be **lost**.

If you need this pull request to be removed from the queue, please contact a
member of the admin team now.




*The following options are set:* **approve, after_pull_request**

bert-e commented:
# 


I have successfully merged the changeset of this pull request
into targetted development branches:


* :heavy_check_mark: `development/2.4`

* :heavy_check_mark: `development/2.5`

* :heavy_check_mark: `development/2.6`



The following branches have **NOT** changed:

* `development/1.0`
* `development/1.1`
* `development/1.2`
* `development/1.3`
* `development/2.0`
* `development/2.1`
* `development/2.2`
* `development/2.3`



Please check the status of the associated issue None.

Goodbye gdemonet.
